app/controllers/concerns/membership_actions.rb

```diff
@@ -16,9 +16,13 @@ module MembershipActions # rubocop:disable Metrics/ModuleLength
   def index
     authorize! @namespace, to: :member_listing?
     respond_to do |format|
```

Comment:
We can set the @q, default sort, and the @pagy, @members stuff before the respond_to block and then within it all you would require is format.html and format.turbo_stream. When setting @q you can just use `@q = load_members.ransack(params[:q])`

The `load_members` method also needs to be updated to convert the array into an activerecord collection so that it will work with ransack. This can be accomplished with `MyModel.where(id: arr.map(&:id))`

Reply:
As discussed we don't require this and can just update the member_policy scope

Reply:
53a6024210cf0bc7d16da260fed1a23a5f3a4a1c